src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -36,7 +36,7 @@ public KeysController(WebHostSettings settings, ISecretManager secretManager, Tr
             _settings = settings;
             _secretManager = secretManager;
             _traceWriter = traceWriter.WithDefaults($"{ScriptConstants.TraceSourceSecretManagement}.Api");
-            _logger = loggerFactory?.CreateLogger(ScriptConstants.LogCategoryKeysController);
+            _logger = loggerFactory?.CreateLogger(ScriptConstants.LogCategoryKeys);
```

Comment:
`KeysController` is the right category for this. `LogCategoryKeys` should be used in the manager/repositories